### PR TITLE
add default expm() using conversion to and from Matrix type

### PR DESCRIPTION
**Pull request by cdsousa**
My editor removed trailing spaces in lots of lines from runtests.jl, is that OK?

### CODE REVIEW

**Comment by SimonDanisch** [comment]
No that's more than welcome! In fact I plan to be a little more pedantic about it, but my current editor set-up doesn't make it easy ;)

**Comment by cdsousa** [comment]
I had ran the test locally and they had passed... Can you help me understanding the problem? I could not figure it out from Travis log...

File: test/runtests.jl [hunk 1]
```diff
@@ -453,18 +453,16 @@ context("Matrix Math") do
...
-			if i <= 2
-			    context("expm(M)") do
-			        mm = expm(m)
-				    fmm = expm(mfs)
-				    @fact isapprox(fmm, mm)  --> true
-
-				    mm = expm(mc)
-				    fmm = expm(mfsc)
-				    @fact isapprox(fmm, mm)  --> true
-			    end
```
Comment (cdsousa) [inline test/runtests.jl]: (Please notice that between the trailing spaces removals there is an actual change to code here.)

File: test/runtests.jl [hunk 2]
```diff
@@ -453,18 +453,16 @@ context("Matrix Math") do
...
+			context("expm(M)") do
+				mm = expm(m)
+				fmm = expm(mfs)
+				@fact isapprox(fmm, mm)  --> true
+
+				mm = expm(mc)
+				fmm = expm(mfsc)
+				@fact isapprox(fmm, mm)  --> true
```
Comment (cdsousa) [inline test/runtests.jl:463]: (Please notice that between the trailing spaces removals there is an actual change to code here.)

**Comment by cdsousa** [comment]
Well, although I've made this PR, I'm now wondering if this kind of falling back to `Array` methods belongs here in this package:
It may be possible that this misleads people into thinking that if the method exists for FSAs then it has a fast implementation, although it is well probable that in many cases using a FSA type and a calling a method with fallback to Arrays is slower than just using Arrays in the first place...

Do you have any policy on this?

**Comment by mschauer** [comment]
maybe https://github.com/JuliaLang/julia/commit/85f45974a581ab9af955bac600b90d9ab00f093b ?

**Comment by SimonDanisch** [comment]
Its probably unrelated to your addition and just a change in Julia. I look into it.

**Comment by SimonDanisch** [comment]
@mschauer is right... And I'm not sure how to fix this. I commented on JuliaLang/julia@85f4597 to see whats up

**Comment by SimonDanisch** [comment]
I think array fallbacks are okay, since when you decide to use FSA and really need e.g. `expm`, its better than nothing. They should be documented though ;)

**Pull request merged**